examples/resources/grafana_cloud_private_datasource_connect/resource.tf

```
@@ -0,0 +1,17 @@
+data "grafana_cloud_stack" "current" {
+  stackID = "<your stack ID>"
+}
+
+resource "grafana_cloud_private_datasource_connect" "test" {
+  region       = "us"
+  name         = "my-pdc"
+  display_name = "My PDC"
+  identifier   = data.grafana_cloud_stack.current.stackID
```

I'm not sure of conventions here, but I'd prefer if this name was more explicit:
```suggestion
  stack_identifier   = data.grafana_cloud_stack.current.stackID
```

---

Done!